transport/kubernetes-jobmonitor/src/main/kotlin/JobHandler.kt

```diff
@@ -79,6 +79,13 @@ internal class JobHandler(
         /** A prefix for the name of a label storing a part of the trace ID. */
         private const val TRACE_LABEL_PREFIX = "trace-id-"
 
+        /**
+         * A label selector to find only jobs for ORT Server components. Only those are handled when looking for
+         * completed or failed jobs.
+         */
+        private val workerJobsLabelSelector = "ort-worker in " +
```

Comment:
Would it make sense to introduce a constant for "ort-worker in ", similar to the above constants?

Reply:
This property is already (semantically) a constant that contains the label selector. I don't think it is an improvement to extract the prefix to another constant.